pegasus/sites.v3/code.org/views/athome_videos.md.erb

```diff
@@ -0,0 +1,27 @@
+<div class="col-33" style="padding-right: 20px;">
+
+<%= view :athome_vid_howcomputers %>
```

Comment:
note that this template could be written as a `.md.partial`, with a simple change to the two view statements:
```suggestion
{{ athome_vid_howcomputers }}
```

Reply:
Ah, nice.